src/app/config/navigation.ts

```diff
@@ -10,9 +16,46 @@ export const stackNavigatorConfig = {
     letterSpacing: -0.4,
     marginBottom: 20
   },
+  headerLeftContainerStyle: {
+    marginBottom: 20
+  },
   headerStyle: NAV_HEIGHT,
   headerBackTitleVisible: false,
   headerBackground: NavBackground,
   headerTransparent: true,
   headerTintColor: WHITE
 };
+
+const libraryIcon = ({ focused }: { focused: boolean }) => TabIcon({ focused, source: libraryImage });
+export const libraryStackConfig = {
+  title: 'Library',
+  tabBarIcon: libraryIcon
+};
+
+const wishlistIcon = ({ focused }: { focused: boolean }) => TabIcon({ focused, source: wishlistImage });
+export const wishlistStackConfig = {
+  title: 'Wishlist',
+  tabBarIcon: wishlistIcon
+};
+
+const addNewIcon = ({ focused }: { focused: boolean }) => TabIcon({ focused, source: addNewImage });
+export const addNewStackConfig = {
+  title: 'Add New',
+  tabBarIcon: addNewIcon
+};
+
+const rentalsIcon = ({ focused }: { focused: boolean }) => TabIcon({ focused, source: rentalsImage });
+export const rentalsStackConfig = {
+  title: 'Rentals',
+  tabBarIcon: rentalsIcon
+};
+
+const settingsIcon = ({ focused }: { focused: boolean }) => TabIcon({ focused, source: settingsImage });
+export const settingsStackConfig = {
+  title: 'Settings',
+  tabBarIcon: settingsIcon
+};
```

Comment:
You are repeating the Icon declaration, you can use directly `screenOptions={{ tabBarIcon: TabBarIcon }}` in the `TabNavigator.Navigator` props
Keep only the `routeStackConfig`:
```
export const libraryStackConfig =  { title: 'Library' };
export const wishlistStackConfig =  { title: 'Wishlist' };
export const addNewStackConfig =  { title: 'Add New' };
export const rentalsStackConfig =  { title: 'Rentals' };
export const settingsStackConfig =  { title: 'Settings' };
```